src/opt-out.html

```diff
@@ -56,6 +56,26 @@ <h2 class="govuk-heading-m">
       <p class="govuk-body">
         If this does not work, contact your device manufacturer.
       </p>
+      <h2 class="govuk-heading-s">
+        Opt out of mobile phone network tests
+      </h2>
+      <p class="govuk-body">
+        Mobile network operators in the UK are testing emergency alerts.
+      </p>
+      <p class="govuk-body">
+        To switch these alerts off or on:
+      </p>
+      <ul class="govuk-list govuk-list--number">
+        <li>
+          Open your phone calling app
+        </li>
+        <li>
+          Use the keypad to enter <span class="extra-letter-spacing govuk-!-font-weight-bold">*#*#2627#*#*</span>
```

Comment:
AFAIK the operator channel is only hidden on devices with Android 11+

Do we know what proportion of users this is likely to be?

Do we need to say anything different for users with an older OS?

Reply:
So far today 37% of Android users visiting GOV.UK are on version 11